docker/image/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:xenial
+
+ARG user=rosindex
+ARG uid=1000
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN echo "deb http://packages.ros.org/ros/ubuntu xenial main" > \
+         /etc/apt/sources.list.d/ros-latest.list
+RUN apt-key adv --keyserver hkp://pool.sks-keyservers.net --recv-key 0xB01FA116
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        apt-utils \
+        build-essential \
+        cmake \
+        curl \
+        git-all \
+        git-svn \
+        libgit2-dev \
+        libpthread-stubs0-dev \
+        libssl-dev \
+        libsslcommon2-dev \
+        libz-dev \
+        mercurial \
+        nodejs \
+        openssl \
+        pandoc \
+        pkg-config \
+        python-pip \
+        python3-vcstool \
+        ruby \
+        ruby-dev \
+        rubygems-integration && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN gem install bundle
+RUN pip install --upgrade setuptools pip
+RUN pip install recommonmark==0.4 sphinx==1.5.6
+
+RUN ln -s `which nodejs` /usr/local/bin/node
+
+COPY docker/image/build_site.sh /usr/local/bin/build_site
+COPY docker/image/test_site.sh /usr/local/bin/test_site
+COPY docker/image/update_site.sh /usr/local/bin/update_site
+
+RUN useradd -u $uid -m $user
+ENV HOME=/home/$user
+WORKDIR $HOME
+COPY Gemfile* .bundle/
+RUN chown -R $user:$user .
+
+USER $user
+ENV BUNDLE_PATH=$HOME/.bundle
+ENV BUNDLE_GEMFILE=$HOME/.bundle/Gemfile
+RUN bundle install
+
+RUN git config --global user.name "rosindex"
+RUN git config --global user.email "rosindex@build.ros.org"
+
+# The environment variables REPO and SITE should be set when running this container.
+# REPO: The path to this source repository in the container
+# SITE: The path to the destination repository in the container
+ENV REPO
+ENV SITE
```

Comment:
This fails for me with `ENV must have two arguments`, which I believe to be this issue: https://github.com/moby/moby/issues/34806.

This works for me:

```diff
diff --git a/docker/image/Dockerfile b/docker/image/Dockerfile
index 241a7c0..c8305fc 100644
--- a/docker/image/Dockerfile
+++ b/docker/image/Dockerfile
@@ -61,6 +61,6 @@ RUN git config --global user.email "rosindex@build.ros.org"
 # The environment variables REPO and SITE should be set when running this container.
 # REPO: The path to this source repository in the container
 # SITE: The path to the destination repository in the container
-ENV REPO
-ENV SITE
+ENV REPO ""
+ENV SITE ""
 CMD update_site ${REPO} ${SITE}
```

Reply:
Hmm, that's funny. Fixed it, and thanks for bringing it up!